cobrawap/pipeline/configs/config_template.yaml

```
@@ -19,8 +19,8 @@ NEO_FORMAT: 'nix'
 USE_LINK_AS_STAGE_OUTPUT: True
 
 # Plotting parameters
-PLOT_TSTART: 0  # in s
-PLOT_TSTOP: 10  # in s
+PLOT_TSTART: 0  # float (in s) or 'None' -> T_START of the input signal is used
+PLOT_TSTOP: 10  # float (in s) or 'None' -> T_STOP of the input signal is used
```

Here and in the following, does `T_START` refer to `t_start` of the Neo object? In that case I would suggest
```suggestion
PLOT_TSTART: 0  # float (in s) or 'None' -> t_start of the input signal is used
PLOT_TSTOP: 10  # float (in s) or 'None' -> t_stop of the input signal is used
```

---

Ok, I will fix that everywhere!